CryptoExchange.Net/CryptoExchange.Net.csproj

```diff
@@ -37,30 +38,16 @@
     <EmbedUntrackedSources>true</EmbedUntrackedSources>
     <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
   </PropertyGroup>
-  <ItemGroup Label="Deterministic Build" Condition="'$(Configuration)' == 'Release'">
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="8.0.0">
```

Comment:
The source link support is already in the SDK since .NET 8.
https://learn.microsoft.com/en-us/dotnet/core/compatibility/sdk/8.0/source-link